c++/src/kj/async-queue.h

```diff
@@ -41,6 +41,12 @@ class WaiterQueue {
   WaiterQueue() = default;
   KJ_DISALLOW_COPY_AND_MOVE(WaiterQueue);
 
+  ~WaiterQueue() {
```

Comment:
By KJ convention this should be:

```suggestion
  ~WaiterQueue() noexcept(false) {
```